tools/filelife.py

```diff
@@ -16,6 +16,8 @@
 #
 # 08-Feb-2015   Brendan Gregg   Created this.
 # 17-Feb-2016   Allan McAleavy updated for BPF_PERF_OUTPUT
+# 13-Nov-2022   Rong Tao        Update vfs_create/vfs_unlink argument in CO-RE
+#                               way, and add vfs_open for f_mode=FMODE_CREATED
```

Comment:
This patch is for bcc tools which do not need CO-RE, so please remove reference to CO-RE in the above.
Could you explain a little bit more in the commit message for how f_mode=FMODE_CREATE related to vfs_open?

Please also change subject line to remove CO-RE reference.